### PR TITLE
Fix: role added to component__widget div (fixes #145)

### LINKED ISSUE #145: aria-labelledby/label not being announced when tabbing through.
### Subject of the issue/enhancement/features
When you tab through a page of questions, the screen reader will announce the header/ariaQuestion for mcq/gmcq but doesn't with the textInput. 

### Your environment
* FW v5.40.3
* Latest Firefox
* Windows, NVDA

### Steps to reproduce
Create a course and use the screen reader to tab through the page.

### Expected behaviour
As with MCQ/GMCQ the header should be read out

### Actual behaviour
The header is not included and only the prefix/text input is announced

### PR DESCRIPTION
**Pull request by joe-allen-89**
[//]: # (Please title your PR according to eslint commit conventions)
[//]: # (See https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-eslint#eslint-convention for details)

[//]: # (Link the PR to the original issue)
#145 

[//]: # (Delete Fix, Update, New and/or Breaking sections as appropriate)
### Fix
* role added to component__widget div (fixes #145)

References:
* https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/group_role
* https://github.com/adaptlearning/adapt-contrib-mcq/blob/652de564338f5e0a2181d3f5f51c17d111cae998/templates/mcq.jsx#L40

### CODE REVIEW

**Review by oliverfoster — APPROVED** [review]

File: templates/textinput.jsx [hunk 1]
```diff
@@ -30,6 +30,7 @@ export default function TextInput (props) {
         ])}
         aria-labelledby={ariaQuestion ? null : (displayTitle || body || instruction) && `${_id}-header`}
         aria-label={ariaQuestion || null}
+        role='group'
```
Comment (oliverfoster) [inline templates/textinput.jsx:33]: Re group vs region see https://github.com/adaptlearning/adapt-contrib-confidenceSlider/pull/113#pullrequestreview-2177068396

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by kirsty-hames — APPROVED** [review]
Works as expected when testing with VoiceOver on Mac 👍

File: templates/textinput.jsx (hunk 1 above)
Comment (joe-allen-89) [reply templates/textinput.jsx:33]: Confidence slider updated to group for consistency which resolves this query.

**Pull request merged**